handlers/deposit_resource.go

```diff
@@ -58,6 +59,7 @@ func (d *depositResource) Handle(params operations.DepositResourceParams, agent
 		WithID(uuid).
 		WithExternalIdentifier(externalID).
 		WithVersion(1)
+	(*resource.Administrative())["created"] = time.Now().UTC().Format(time.RFC3339)
```

Comment:
@eefahy did you / we ever move the ball on capturing or using the dynamodb datetime created for the sake of this admin field? i think you said it was captured, it was just a question of putting it in the metadata or resource responses for sake of clients + users.

Reply:
never mind, dynamodb date is unreliable.